src/pages/Home/Home.tsx

```diff
@@ -1,19 +1,33 @@
 import React from 'react'
 import Button from '@mui/material/Button'
 import SendIcon from '@mui/icons-material/Send';
+import { useDispatch } from 'react-redux';
 import styles from './home.module.scss'
+import { useFetchLogoutMutation } from '../../services/AuthServices';
+import { setCredentials } from '../../store/reducers/AuthSlice';
+import { RouterLinksName } from '../../utils/consts';
 
-const Home:React.FC = () => (
-  <>
-    <div className={styles.active}>Home</div>
-    <Button
-      variant='contained'
-      color='success'
-      endIcon={<SendIcon />}
-    >
-      This is my home!
-    </Button>
-  </>
-)
+const Home:React.FC = () => {
+  const [fetchLogout] = useFetchLogoutMutation();
+  const dispatch = useDispatch();
```

Comment:
❗️
Кажется, что на проекте уже есть типизированный диспатч
```suggestion
  const dispatch = useAppDispatch();
```